### ISSUE TITLE
[Color Picker] Add a new `label` property to specify a label with the component (CAL-AMP58)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#13545]
**monday.com sync:** #10821964185

### Violation: 
 Ensure headings and labels are descriptive and unique 

### Recommended solution
Add a new `label` property to the component so apps and solutions can identify the purpose of the component. _Discussion is needed_ to determine if the property should be required.

### WCAG Reference: 
- [2.4.6 Headings and Labels](https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels.html) 

### Severity: 
 Low 

### Media Type: 
 Page Structure 

--- 
### Areas for Remediation: 
- Title: CAL-AMP58 Ensure headings and labels are descriptive and unique 
- Module: M7 Table and color picker 
- Finding ID: [CAL-AMP58](https://esri.hub.essentia11y.com/short-link/SUr2LHSoBP3P5HZb) 

**Issue**
There is a "Value" slider with an insufficiently descriptive accessible name. 

**User Impact**
Screen reader users will be unable to determine the purpose of these buttons.

**Code Reference** 
 ```
<div aria-valuemin="0" role="slider" tabindex="0" aria-label="Value" aria-valuemax="100" aria-valuenow="71.2121212121212" class=" scope scope--color-field " style="top: 97.3353px; left: 168.985px;"></div>

```

**Steps to Reproduce**
1. Locate the button.
2. Inspect it with Chrome DevTools.
3. In the Accessibility tab, expand the Computed Properties section.
4. Review the value for "Name".
 
**Recommendation**
Ensure headings and labels are descriptive and unique. This includes the accessible names of buttons. The accessible name of a slider can be set with internal text, an aria-label attribute, or an aria-labelledby attribute. Good accessible names are both concise and descriptive. Avoid including the word "slider" in the accessible name, as this information is already supplied by the button's role.

In this case, the developer should provide a descriptive accessible name as "Saturation value".

**Compliant Code Example**
```
<!-- example of headings that describe topic or purpose -->
<h1>Dashboard</h1> 
<h2>Reporting</h2>
<h3>Reporting Details </h3>
<h2>Testing</h2> 
<h3>Testing Details</h3>
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP58.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/SUr2LHSoBP3P5HZb) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.
